src/lottery-application.ts

```diff
@@ -47,7 +48,8 @@ class Lottery {
             count++;
         } while (count < this.numbersToDraw);
         totalDurationStopWatch.stopWatch();
-        this.totalDuration = totalDurationStopWatch.getDuration();
+        const totalDurationEndCount = totalDurationStopWatch.end;
```

Comment:
This is not ok. The `getDuration()` method is intended to return the number of milliseconds between the start and end. Exposing the internal state of `StopWatch` and delegating this logic to the client isn't a good idea.